data/store/chatStore.ts

```diff
@@ -191,15 +74,12 @@ export const initChatStore = (account: string) => {
     persist(
       (set) =>
         ({
-          conversations: {},
```

Comment:
just so i am tracking this was moved to be handled by react query?

Reply:
Exactly! the goal would be to erase ChatStore I think. So as soon as I'm moving stuff away I'm trying to clean from ChatStore

Reply:
Yes! A chat store is fine, but the previous chat store was resulting in many things updating too often or requiring manual usage